test/integration/flex.integration.test.ts

```diff
@@ -146,7 +150,14 @@ describe("Flex Integration Test", () => {
         expect(result.status == 200).toBeTruthy();
     }, 15000);
 
-
-    
-
+    test('Verifying the service can access File Storage with file URL ', async () => {
```

Comment:
Can we try listing the file of flex container , this validates container exists and connection establishment is complete.